app/helpers/admin/enterprises_helper.rb

```diff
@@ -22,6 +22,14 @@ def enterprise_side_menu_items(enterprise)
       show_enterprise_fees = can?(:manage_enterprise_fees,
                                   enterprise) && (is_shop || enterprise.is_primary_producer)
 
+      build_enterprise_side_menu_items(is_shop, show_properties, show_shipping_methods,
+                                       show_payment_methods, show_enterprise_fees)
+    end
+
+    private
+
+    def build_enterprise_side_menu_items(is_shop, show_properties, show_shipping_methods,
+                                         show_payment_methods, show_enterprise_fees)
```

Comment:
👍 
I've often wondered how we can arrange this better. I wondered if it could be moved to an erb template... But there are more important things to wonder about!